src/styles/blocks/place.scss

```diff
@@ -0,0 +1,67 @@
+.place {
+  &__photo-container {
+    position: relative;
+
+    width: 100%;
+    box-sizing: border-box;
+    max-height: 320px;
+    overflow: hidden;
+
+    &:hover {
+      transform: scale(105%);
+      transition: transform 0.3s;
```

Comment:
move transition to &__photo-container class to make smooth animation in both ways